cmd/podman/images/pull.go

```diff
@@ -154,15 +154,20 @@ func imagePull(cmd *cobra.Command, args []string) error {
 	}
 	// Let's do all the remaining Yoga in the API to prevent us from
 	// scattering logic across (too) many parts of the code.
-	pullReport, err := registry.ImageEngine().Pull(registry.GetContext(), args[0], pullOptions.ImagePullOptions)
-	if err != nil {
-		return err
+	pulledImages := make([]string, len(args))
+	for i, arg := range args {
+		pullReport, err := registry.ImageEngine().Pull(registry.GetContext(), arg, pullOptions.ImagePullOptions)
+		if err != nil {
+			return err
+		}
+		pulledImages[i] = pullReport.Images[0]
+		fmt.Print("\n")
```

Comment:
I assume the newline was added to have a blank line between each image pull but it will break `-q`.  I suggest to just remove the line.

Reply:
Sure Ill remove it, but can we not do something like:
```go
if !pullOptions.Quiet {
    fmt.Print("\n")
}
```

Reply:
The output of `pull --all foo` doesn't have these blank lines, so it would not be consistent.  But I do not feel strongly.  If you like the blank lines, feel free to add them.